modules/ROOT/pages/endpoint-https-receive.adoc

```diff
@@ -40,19 +48,18 @@ Possible values include:
 |Yes
 
 |Basic authentication
-a|When Basic authentication is selected, specify the user name and password to protect the HTTP listener.
+a|When Basic authentication selected, specify the user name and password to protect the HTTP listener.
 
-You need to note the password specified during the creation of the endpoint. After the configuration is saved, you cannot retrieve or modify the password.
+After the configuration or creation of the endpoint is saved, you cannot retrieve or modify the password.
```

Comment:
configuration or creation is saved?
Hmm...can we say only "After your new or [re?]configured endpoint is saved, you cannot subsequently..."?

Reply:
Done!